source/PreferencesPanel.cpp

```diff
@@ -710,10 +714,20 @@ void PreferencesPanel::DrawSettings()
 		}
 		else if(setting == EXPEND_AMMO)
 			text = Preferences::AmmoUsage();
+		else if(setting == DATE_FORMAT)
+		{
+			isOn = true;
+			Preferences::DateFormat datefmtShortName = Preferences::GetDateFormat();
+			if(datefmtShortName == Preferences::DateFormat::ymd)
+				text = "yyyy-mm-dd";
+			else if(datefmtShortName == Preferences::DateFormat::dmy)
+				text = "dd/mm/yyyy";
+			else if(datefmtShortName == Preferences::DateFormat::mdy)
+				text = "mm/dd/yyyy";
```

Comment:
When tehhowch said "The strings in this array should be used in the preferences panel (see vsync)", this is what he meant. Notably, this DateFormatSetting function is currently unused.
```suggestion
			text = Preferences::DateFormatSetting();
			isOn = true;
```